lib/updatehub/updatehub.c

```diff
@@ -383,13 +452,40 @@ static void install_update_cb(void)
 		LOG_ERR("Could not receive data");
 		goto cleanup;
 	}
-
```

Comment:
Having empty line here is good, please remove this change.

Reply:
done